src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -16,6 +17,25 @@ public partial class PackageDependencyView : UserControl
 
         private WorkspaceModel currentWorkspace;
 
+        private ViewLoadedParams loadedParams;
+        private PackageDependencyViewExtension dependencyViewExtension;
+
+        private Boolean missingPackage = false;
+
+        public Boolean MissingPackage
```

Comment:
also, please add a summary for a public property - does it need to be public?

Reply:
Seem internal is enough

Reply:
I will be removing the check from PackageDependencyViewExtension.cs as it is not needed. So this can be private. Will add the summary